src/test/java/pageObjects/DevicePageTests.java

```diff
@@ -99,13 +204,53 @@ public void createDeviceWithTooLongInventoryNo() {
         Assert.assertEquals(errorToaster.getText(), "The field \"inventoryNumber\" is too long.");
     }
 
+    @Test
+    public void editDeviceTooLongInventoryNo() {
+        devicePage.editDeviceTooLongInventoryNo();
+        WebElement errorToaster = driver.findElement(By.id("error_toaster"));
+        Assert.assertEquals(errorToaster.getText(), "The field \"inventoryNumber\" is too long.");
+    }
+
     @Test
     public void createDeviceWithTooLongInvoiceNo() {
         devicePage.createDeviceTooLongInvoiceNo();
         WebElement errorToaster = driver.findElement(By.id("error_toaster"));
         Assert.assertEquals(errorToaster.getText(), "The field \"invoiceNumber\" is too long.");
     }
 
+    @Test
+    public void editDeviceTooLongInvoiceNo(){
+        devicePage.editDeviceTooLongInvoiceNo();
+        WebElement errorToaster = driver.findElement(By.id("error_toaster"));
+        Assert.assertEquals(errorToaster.getText(), "The field \"invoiceNumber\" is too long.");
+    }
+
+//    @Test
+//    public void successfullyEditInvoiceDate(){
+//        devicePage.successfullyEditInvoiceDate();
+//        Assert.assertEquals(devicePage.successEdit, "Item updated successfully!");
+//        String field = driver.findElement(By.id("device-table")).getText();
+//        Assert.assertTrue(field.contains(Constants.INVOICE_DATE));
+//    }
+
+//    @Test
+//    public void successfullyEditStartDateAndEndDate(){
+//        devicePage.successfullyEditStartDateAndEndDate();
+//        String successEdit = driver.findElement(By.id("success_toaster")).getText();
+//        Assert.assertEquals(successEdit, "Item updated successfully!");
+//        String editedDevice =driver.findElement(By.id("device-table")).getText();
+//        Assert.assertTrue(editedDevice.contains(devicePage.getDeviceWarrantyStartDate()));
+//        Assert.assertTrue(editedDevice.contains(devicePage.getDeviceWarrantyEndDate()));
+//    }
```

Comment:
you can leave both the test and method uncommented and just comment the @Test annotation so it will not run